migrations/1605637330716-CreateMakerBalanceChainCache.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+const makerBalanceChainCacheTable = new Table({
+    name: 'maker_balance_chain_cache',
+    columns: [
+        { name: 'token_address', type: 'varchar', isPrimary: true },
+        { name: 'maker_address', type: 'varchar', isPrimary: true },
+        // fields are nullable for the job that adds new rows upon
+        // discovery of a new maker address
+        { name: 'balance', type: 'varchar', isNullable: true },
```

Comment:
oh interesting, so we are keeping this as `varchar`?

Reply:
see other comment.